### DISCUSSION TITLE
What is going to happen to either uno or windi

### DISCUSSION THREAD

**Discussion by louiss0** [unocss/unocss#1969] [Q&A]
I'm concerned about the fact that Windi has not been updated in a long time. The color palette has not changed. UnoCSS is it's competitor. I don't think it makes sense to keep them both alive. I don't know whether or not someone is going to have the time to focus on either project. I see very few updates to either one. While tailwind just keeps adding new features. There are lots of tools for a tailwind but you have to pay for most of them. There is a google extension that even allows you to pick colors and tells you not it is compatible with tailwindcss. What is the plan for the two projects

**Comment by awacode21** [comment]
As far as i read WindiCSS is officially sunsetting. So it seems they concentrate on unocss now.
